internal/state/graph/httproute_test.go

```diff
@@ -298,14 +221,13 @@ func TestBuildSectionNameRefs(t *testing.T) {
 	}
 
 	gwNsNames := []types.NamespacedName{gwNsName1, gwNsName2}
-	routeNamespace := "test"
 
-	expected := map[string]ParentRef{
-		"one": {
+	expected := []ParentRef{
```

Comment:
Is there a test for the case where there are parentRefs with the same section name but different gateways?